Creating a blank disk image (DV/PVC)

ousleyp opened the pull request:
@openshift/team-documentation PTAL <3

mburke5678 commented:
@ousleyp A couple of pesky questions. Otherwise LGTM

ousleyp commented:
> @ousleyp A couple of pesky questions. Otherwise LGTM

Where can I find the questions? :)

mburke5678 commented:
@ousleyp Sorry. I think I added the comments. i was commenting via my smart phone for the first time. Apparently I didn't complete the process.

ousleyp commented on modules/cnv_blankdisk.adoc:
```diff
@@ -0,0 +1,60 @@
+.Prerequisites
+
+* {ProductName} 1.4
+* At least one available PV
```
I'll spell it out - good call :)

ousleyp commented on modules/cnv_blankdisk.adoc:
```diff
@@ -0,0 +1,60 @@
+.Procedure
+
+. Create the manifest file.
+* To use a DataVolume manifest, create `blank-image-datavolume.yaml`:
```
I don't think it's required (on either count). I'll take the filename out of the first step.

ousleyp commented:
> @ousleyp Sorry. I think I added the comments. i was commenting via my smart phone for the first time. Apparently I didn't complete the process.

Thanks for the review, mburke! No worries at all. Please check out the changes I made and let me know what you think. I also split it up into two procedures based on other feedback.

vikram-redhat commented:
/cherrypick enterprise-3.11
